### PR TITLE
FAT-9320 Filter request by tag (Vega)

### PR DESCRIPTION
**Pull request by victoriakor**
FAT-9320
https://foliotest.testrail.io/index.php?/cases/view/9320
[Allure report](https://jenkins-aws.indexdata.com/job/Testing/job/Cypress_tests/4297/allure/)

### CODE REVIEW

File: cypress/e2e/requests/filter-request-by-tags.cy.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+import InventoryInstance from '../../support/fragments/inventory/inventoryInstance';
+import { ITEM_STATUS_NAMES, REQUEST_TYPES } from '../../support/constants';
+
+describe('ui-requests: Filter requests by tags', () => {
+  const userIds = [];
+  const requests = [];
+  const instances = [];
+  const requestTypes = { PAGE: 'Page', HOLD: 'Hold', RECALL: 'Recall' };
+  let oldRulesText;
+  let requestPolicyId;
```
Comment (dnovikau) [inline cypress/e2e/requests/filter-request-by-tags.cy.js:15]: Please remove unused variables

File: cypress/e2e/requests/filter-request-by-tags.cy.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    });
+    requests.forEach((request) => {
+      Requests.deleteRequestViaApi(request.id);
+    });
+    userIds.forEach((id) => {
+      Users.deleteViaApi(id);
+    });
+  });
+
+  it('C9320 Filter requests by tags', { tags: [testType.smoke, DevTeams.vega] }, () => {
```
Comment (dnovikau) [inline cypress/e2e/requests/filter-request-by-tags.cy.js:55]: Test type is not smoke but 'Extended'

Comment (dnovikau) [inline cypress/e2e/requests/filter-request-by-tags.cy.js:55]: Test description should ends with devTeam: " (vega)"

**Review by dnovikau — CHANGES_REQUESTED** [review]
The test is failing on my environment with the error: 
NoSuchElementError
did not find multi select option "[" within multiselect dropdown

File: cypress/e2e/requests/filter-request-by-tags.cy.js (hunk 1 above)
Comment (victoriakor) [reply cypress/e2e/requests/filter-request-by-tags.cy.js:15]: Steps for updating Circulation Rules and Delete Request Policy were added in afterHook so  variables from 14 and 15 rows are necessary now.

File: cypress/e2e/requests/filter-request-by-tags.cy.js (hunk 2 above)
Comment (victoriakor) [reply cypress/e2e/requests/filter-request-by-tags.cy.js]: done, thank you

Comment (victoriakor) [reply cypress/e2e/requests/filter-request-by-tags.cy.js]: done, thank you

File: cypress/e2e/requests/filter-request-by-tags.cy.js [hunk 3]
```diff
@@ -50,9 +50,11 @@ describe('ui-requests: Filter requests by tags', () => {
     userIds.forEach((id) => {
       Users.deleteViaApi(id);
     });
+    Requests.updateCirculationRulesApi(oldRulesText);
```
Comment (dnovikau) [inline cypress/e2e/requests/filter-request-by-tags.cy.js:74]: Looks like it is redundant

File: cypress/e2e/requests/filter-request-by-tags.cy.js [hunk 4]
```diff
@@ -50,9 +50,11 @@ describe('ui-requests: Filter requests by tags', () => {
...
+    Requests.deleteRequestPolicyApi(requestPolicyId);
```
Comment (dnovikau) [inline cypress/e2e/requests/filter-request-by-tags.cy.js:75]: Redundant

Comment (victoriakor) [reply cypress/e2e/requests/filter-request-by-tags.cy.js:75]: Done.- Removed unnecessary steps.

File: cypress/e2e/requests/filter-request-by-tags.cy.js (hunk 3 above)
Comment (victoriakor) [reply cypress/e2e/requests/filter-request-by-tags.cy.js]: Done)

**Review by dnovikau — APPROVED** [review]

**Review by TetianaParanich — APPROVED** [review]

**Pull request merged**